### PR TITLE
Update Japanese translations

### PR DESCRIPTION
**Pull request by tkusano**
* Updated as PR on Plural is merged (#3882)
* Feedback API (#3498)

etc.

### CODE REVIEW

File: src/locale/locales/ja/messages.po [hunk 1]
```diff
@@ -17,19 +17,107 @@ msgstr ""
 msgid "(no email)"
 msgstr "メールがありません"
 
+#: src/view/com/notifications/FeedItem.tsx:239
+msgid "{0, plural, one {{formattedCount} other} other {{formattedCount} others}}"
+msgstr "{0, plural, other {他{formattedCount}人}}"
+
+#: src/components/moderation/LabelsOnMe.tsx:57
+msgid "{0, plural, one {# label has been placed on this account} other {# labels has been placed on this account}}"
+msgstr "{0, plural, other {#個のラベルがこのアカウントに貼られています}}"
```
Comment (dolciss) [inline src/locale/locales/ja/messages.po:26]: 英文の時点でplacedとappliedが混在しているのですが、ラベルが貼られているor適用されているで揺れているため統一できるとうれしいです。
```suggestion
msgstr "{0, plural, other {#個のラベルがこのアカウントに適用されています}}"
```

File: src/locale/locales/ja/messages.po [hunk 2]
```diff
@@ -17,19 +17,107 @@ msgstr ""
...
+
+#: src/components/moderation/LabelsOnMe.tsx:63
+msgid "{0, plural, one {# label has been placed on this content} other {# labels has been placed on this content}}"
+msgstr "{0, plural, other {#個のラベルがこのコンテンツに貼られています}}"
```
Comment (dolciss) [inline src/locale/locales/ja/messages.po:30]:
```suggestion
msgstr "{0, plural, other {#個のラベルがこのコンテンツに適用されています}}"
```
同様の箇所が以下の行にあります（"ラベル"でgrep）
貼る：546, 554, 5650
適用：3388, 4688, 4692, 4881, 4885,

**Review by dolciss — COMMENTED** [review]
Thank you for always submitting PRs for translations and for handling the plural form of #3382.
Please confirm my comment on the labeling notation shaking.

File: src/locale/locales/ja/messages.po (hunk 2 above)
Comment (tkusano) [reply src/locale/locales/ja/messages.po]: すべて「適用」に統一してみました

**Comment by dolciss** [comment]
Thanks for applying your comment!
LGTM

**Comment by tkusano** [comment]
Updated translations related to following PRs
- #3934 
- #3937 
- #3913 
- #3921

File: src/locale/locales/ja/messages.po [hunk 3]
```diff
@@ -19,69 +19,73 @@ msgstr "メールがありません"
...
 msgid "{0, plural, one {repost} other {reposts}}"
-msgstr ""
+msgstr "{0, plural, other {リポスト}}"
 
 #: src/view/com/util/post-ctrls/PostCtrls.tsx:241
 msgid "{0, plural, one {Unlike (# like)} other {Unlike (# likes)}}"
+msgstr "{0, plural, other {いいねを外す (#個のいいね)}}"
+
+#: src/view/screens/ProfileList.tsx:286
+msgid "{0} your feeds"
```
Comment (tkusano) [inline src/locale/locales/ja/messages.po:75]: I'll leave this message untranslated until #3962  is resolved

**Comment by Hima-Zinn** [comment]
I will check this PR tomorrow.

File: src/locale/locales/ja/messages.po [hunk 4]
```diff
@@ -4979,17 +5010,17 @@ msgstr "この情報は他のユーザーと共有されません。"
...
 msgstr "これは、メールアドレスの変更やパスワードのリセットが必要な場合に重要です。"
 
-#: src/components/moderation/ModerationDetailsDialog.tsx:124
-#~ msgid "This label was applied by {0}."
-#~ msgstr "{0}によって適用されたラベルです。"
-
 #: src/components/moderation/ModerationDetailsDialog.tsx:127
 msgid "This label was applied by <0>{0}</0>."
-msgstr ""
+msgstr "<0>{0}</0>よって適用されたラベルです。"
```
Comment (middlingphys) [inline src/locale/locales/ja/messages.po:5015]:
```suggestion
msgstr "<0>{0}</0>によって適用されたラベルです。"
```

File: src/locale/locales/ja/messages.po [hunk 5]
```diff
@@ -5090,6 +5117,10 @@ msgstr "スレッドの設定"
 msgid "To disable the email 2FA method, please verify your access to the email address."
 msgstr "メールでの２要素認証を無効にするには、メールアドレスにアクセスできるか確認してください。"
 
+#: src/components/dms/ConvoMenu.tsx:202
+msgid "To report a conversation, please report one of its messages via the conversation screen. This lets our moderators understand the context of your issue."
+msgstr "会話を報告するには、会話の画面経由でメッセージを報告してください。モデレーターがあなたの報告の文脈をしやすくなります。"
```
Comment (middlingphys) [inline src/locale/locales/ja/messages.po:5122]:
```suggestion
msgstr "会話を報告するには、会話の画面経由でメッセージを報告してください。モデレーターがあなたの報告の文脈を理解しやすくなります。"
```

**Comment by tkusano** [comment]
@middlingphys Fixed. thanks!

File: src/locale/locales/ja/messages.po [hunk 6]
```diff
@@ -3124,6 +3137,10 @@ msgstr "オンボーディングのリセット"
 msgid "One or more images is missing alt text."
 msgstr "１つもしくは複数の画像にALTテキストがありません。"
 
+#: src/screens/Onboarding/StepProfile/index.tsx:107
+msgid "Only .jpg and .png files are supported"
+msgstr ".jpgと.pngファイルのみに対応してます"
```
Comment (Hima-Zinn) [inline src/locale/locales/ja/messages.po:3142]:
```suggestion
msgstr ".jpgと.pngファイルのみに対応しています"
```

"い"が抜けている。

File: src/locale/locales/ja/messages.po [hunk 7]
```diff
@@ -4979,17 +5015,17 @@ msgstr "この情報は他のユーザーと共有されません。"
...
+msgstr "<0>{0}</0>によって適用されたラベルです。"
 
 #: src/components/moderation/ModerationDetailsDialog.tsx:125
 msgid "This label was applied by the author."
-msgstr ""
+msgstr "投稿者によって適用されたラベルです。"
+
+#: src/components/moderation/LabelsOnMeDialog.tsx:165
+msgid "This label was applied by you"
+msgstr "このラベルはあなたが適用しました"
```
Comment (Hima-Zinn) [inline src/locale/locales/ja/messages.po:5028]:
```suggestion
msgstr "あなたによって適用されたラベルです"
```

上の２つと言い方をそろえる。
ここは最後に"."がないので、変更前が正しいようであればそれでもかまいません。

**Review by Hima-Zinn — COMMENTED** [review]
Please check.

**Comment by pfrazee** [comment]
Looks like there are some unresolved discussions here so Im going to hold off merging. Ping me when this is ready!

Thank you everybody! You rule!

**Comment by Hima-Zinn** [comment]
@tkusano 
Please check my review.
After that, ping pfrazee-san.

File: src/locale/locales/ja/messages.po (hunk 6 above)
Comment (tkusano) [reply src/locale/locales/ja/messages.po]: 修正しました

File: src/locale/locales/ja/messages.po (hunk 7 above)
Comment (tkusano) [reply src/locale/locales/ja/messages.po]: 揃えました

**Comment by tkusano** [comment]
@pfrazee It looks like it didn't make it to 1.82, could you please merge this? Also, I would appreciate it if you could merge #3962 as well, as there is a message that cannot be translated.

**Comment by haileyok** [comment]
@tkusano Got #3962 merged in for you 👍 thank all of you for this work!

**Comment by tkusano** [comment]
@haileyok thanks!

**Comment by tkusano** [comment]
Updated (cf. #4032 )

**Review by pfrazee — APPROVED** [review]
Thanks all!

**Pull request merged**